### PR TITLE
fix(axis): Show min/max split line for all axis types

### PR DESCRIPTION
**Pull request by sebastianludwig**
## Brief Information

This pull request is in the type of:

- [x] bug fixing
- [x] new feature
- [ ] others

### What does this PR do?

Add the missing implementation of `splitLine.showMinLine`/`.showMaxLine` to radius, angle and single axis. Also add support for these properties to `axisLine` for polar angle axis.

### Before: What was the problem?

https://github.com/apache/echarts/pull/20114 added `splitLine.showMinLine` and `.showMaxLine` and implemented it for the cartesian axis. The current documentation shows the option to be supported for other axis as well:

* https://echarts.apache.org/en/option.html#radiusAxis.splitLine.showMinLine
* https://echarts.apache.org/en/option.html#singleAxis.splitLine.showMinLine
* https://echarts.apache.org/en/option.html#angleAxis.splitLine.showMinLine

However there was simply no implementation in place. This PR fixes that.

Also, while on it, I also added support for these properties to `axisLine` for polar angle axis.

#### The split line fix
Screenshot 2026-04-14 at 13 18 06
Screenshot 2026-04-14 at 13 18 29
Screenshot 2026-04-14 at 13 18 50

#### The axis line feature

Screenshot 2026-04-14 at 13 19 44
Screenshot 2026-04-14 at 13 19 59
Screenshot 2026-04-14 at 13 20 15
Screenshot 2026-04-14 at 13 20 29
Screenshot 2026-04-14 at 13 20 44

## Document Info

One of the following should be checked.

- [ ] This PR doesn't relate to document changes
- [ ] The document should be updated later
- [x] The document changes have been made in apache/echarts-doc#505

### Security Checking

- [ ] This PR uses security-sensitive Web APIs.

### ZRender Changes

- [ ] This PR depends on ZRender changes (ecomfe/zrender#xxx).

### Related test cases or examples to use the new APIs

N.A.

### Merging options

- [ ] Please squash the commits into a single one when merging.

### Other information

This is my first PR in this repo. Please let me know if I missed some of the conventions.

### CODE REVIEW

File: src/component/axis/AngleAxisView.ts [hunk 1]
```diff
@@ -138,23 +138,32 @@ interface AngleAxisElementBuilder {
...
-
-        // extent id of the axis radius (r0 and r)
-        const rId = getRadiusIdx(polar);
-        const r0Id = rId ? 0 : 1;
-        let shape;
+        const showMinLine = axisLineModel.get('showMinLine') !== false;
+        const showMaxLine = axisLineModel.get('showMaxLine') !== false;
         const shapeType = Math.abs(angleExtent[1] - angleExtent[0]) === 360 ? 'Circle' : 'Arc';
+        const radiusDataExtent = radiusAxis.scale.getExtent();
+        const minRadius = radiusAxis.dataToCoord(radiusDataExtent[0]);
```
Comment (Justin-ZS) [inline src/component/axis/AngleAxisView.ts]: Potential edge case: for category radius axes, computing boundaries via `radiusAxis.scale.getExtent()` plus `radiusAxis.dataToCoord(...)` may shift min/max line inward because `onBand` affects `dataToCoord`. Could we consider using `radiusAxis.getExtent()` boundaries directly, or handling `onBand` explicitly, so `axisLine.showMinLine/showMaxLine` stays aligned with axis boundaries?

**Comment by Justin-ZS** [comment]
Thanks for the PR. The overall direction looks good, and the `splitLine` coverage extension to `radius`/`angle`/`single` plus added test pages is very helpful. I left one inline comment on `AngleAxisView` for a possible category-axis boundary edge case. Other than that, this looks solid.

**Comment by sebastianludwig** [comment]
Good catch! I've implemented a test case showcasing the problem you foresaw

Screenshot 2026-04-18 at 15 15 19

Next I changed the implementation as suggested to use `radiusAxis.getExtent()` directly and now the chart looks like this/correct:

Screenshot 2026-04-18 at 15 29 31

File: src/component/axis/AngleAxisView.ts (hunk 1 above)
Comment (sebastianludwig) [reply src/component/axis/AngleAxisView.ts]: Changed to use `radiusAxis.getExtent()` 👍

**Review by Justin-ZS — APPROVED** [review]
LGTM

**Comment by gioneves** [comment]
Thanks for working on this! I noticed `radar.splitLine.showMinLine`/`showMaxLine` 
has the same issue — it's listed in the docs and even appears in the component's 
default options (`showMinLine: true, showMaxLine: true`), but setting 
`showMinLine: false` has no visible effect on rendering.

Minimal repro:

\`\`\`javascript
radar: {
  indicator: [
    { name: 'Sales', max: 6500 },
    { name: 'Administration', max: 16000 },
    { name: 'IT', max: 30000 }
  ],
  splitLine: {
    showMinLine: false,
    lineStyle: { width: 2, color: 'red' }
  }
}
\`\`\`

The minimum split line still renders regardless of the `showMinLine` value.

Would it make sense to include `radar` in the scope of this PR as well, since it 
has its own `splitLine` build logic (`_buildSplitLineAndArea`) separate from 
`singleAxis`/`angleAxis`/`radiusAxis`? Happy to help test if useful.
